dmutils/content_loader.py

```diff
@@ -265,50 +264,24 @@ def get_error_messages(self, errors, lot):
         :return: error dictionary with human readable error messages
         """
         errors_map = {}
-        for question_id, message_key in errors.items():
-            field_name = question_id
-            if question_id == 'serviceTypes':
-                field_name = question_id = 'serviceType{}'.format(lot)
-            elif question_id in PRICE_FIELDS:
-                message_key = self._rewrite_pricing_error_key(question_id, message_key)
-                field_name = question_id = 'priceString'
-            elif message_key == 'assurance_required':
-                field_name = '{}--assurance'.format(question_id)
-
-            validation_message = self.get_error_message(question_id, message_key)
-
-            errors_map[field_name] = {
+        for field_name, message_key in errors.items():
+            if field_name == 'serviceTypes':
+                field_name = 'serviceType{}'.format(lot)
```

Comment:
I don't think this is right. We rewrite the `serviceType*` to `serviceType` in `question.id` on import (https://github.com/alphagov/digitalmarketplace-utils/blob/price-formatting-changes/dmutils/content_loader.py#L654), and now that each question is asked to look up the id (as opposed to searching for a match in `section.questions`) I don't think we need the id rewrite here. But `serviceTypes` is confusing, and I might be misunderstanding what happens here.